### PR TITLE
Proper fix for issue 199 - to go into version 2.x

### PR DESCRIPTION
**Pull request by boramonideep**
By creating this pull request you agree to the terms in CONTRIBUTING.md.
https://github.com/Infineon/.github/blob/master/CONTRIBUTING.md
--- DO NOT DELETE ANYTHING ABOVE THIS LINE ---

CONTRIBUTING.md also tells you what to expect in the PR process.

Description
Pin 14 on XMC4 series boards is a VAREF pin and hence cannot be used as a GPIO.

Related Issue
Issue https://github.com/Infineon/XMC-for-Arduino/issues/199

Context
The incorrect pin assignments have been removed and pin configurations have been altered accordingly for all peripherals.

### CODE REVIEW

**Review by techpaul — APPROVED** [review]

File: variants/XMC4700/config/XMC4700_Relax_Kit/pins_arduino.h [hunk 1]
```diff
@@ -277,14 +269,14 @@ XMC_PWM4_t mapping_pwm4[] =
...
-    {CCU42, CCU42_CC42, 2, mapping_port_pin[36], P3_4_AF_CCU42_OUT2, XMC_CCU4_SLICE_PRESCALER_64, PWM4_TIMER_PERIOD, DISABLED},     // PWM disabled  36  P3.4
-    {CCU43, CCU43_CC40, 0, mapping_port_pin[78], P6_5_AF_CCU43_OUT0, XMC_CCU4_SLICE_PRESCALER_64, PWM4_TIMER_PERIOD, DISABLED},     // PWM disabled  78  P6.5
-    {CCU43, CCU43_CC41, 1, mapping_port_pin[90], P6_4_AF_CCU43_OUT1, XMC_CCU4_SLICE_PRESCALER_64, PWM4_TIMER_PERIOD, DISABLED},     // PWM disabled  90  P6.4
-    {CCU43, CCU43_CC43, 3, mapping_port_pin[91], P6_2_AF_CCU43_OUT3, XMC_CCU4_SLICE_PRESCALER_64, PWM4_TIMER_PERIOD, DISABLED}      // PWM disabled  91  P6.2
+    {CCU40, CCU40_CC40, 0, mapping_port_pin[93], P0_15_AF_CCU40_OUT0, XMC_CCU4_SLICE_PRESCALER_64, PWM4_TIMER_PERIOD, DISABLED},    // PWM disabled  93  P0.15
+    {CCU40, CCU40_CC41, 1, mapping_port_pin[70], P0_14_AF_CCU40_OUT1, XMC_CCU4_SLICE_PRESCALER_64, PWM4_TIMER_PERIOD, DISABLED},    // PWM disabled  70  P0.14
+    {CCU40, CCU40_CC43, 3, mapping_port_pin[93], P0_12_AF_CCU40_OUT3, XMC_CCU4_SLICE_PRESCALER_64, PWM4_TIMER_PERIOD, DISABLED},    // PWM disabled  93  P0.12
```
Comment (jaenrig-ifx) [inline variants/XMC4700/config/XMC4700_Relax_Kit/pins_arduino.h:274]:
```suggestion
    {CCU40, CCU40_CC43, 3, mapping_port_pin[94], P0_12_AF_CCU40_OUT3, XMC_CCU4_SLICE_PRESCALER_64, PWM4_TIMER_PERIOD, DISABLED},    // PWM disabled  93  P0.12
```

File: variants/XMC4700/config/XMC4700_Relax_Kit/pins_arduino.h [hunk 2]
```diff
@@ -277,14 +269,14 @@ XMC_PWM4_t mapping_pwm4[] =
...
+    {CCU42, CCU42_CC40, 0, mapping_port_pin[61], P3_0_AF_CCU42_OUT0, XMC_CCU4_SLICE_PRESCALER_64, PWM4_TIMER_PERIOD, DISABLED},     // PWM disabled  61  P3.0
+    {CCU42, CCU42_CC42, 2, mapping_port_pin[33], P3_4_AF_CCU42_OUT2, XMC_CCU4_SLICE_PRESCALER_64, PWM4_TIMER_PERIOD, DISABLED},     // PWM disabled  33  P3.4
```
Comment (jaenrig-ifx) [inline variants/XMC4700/config/XMC4700_Relax_Kit/pins_arduino.h:276]:
```suggestion
    {CCU42, CCU42_CC42, 2, mapping_port_pin[34], P3_4_AF_CCU42_OUT2, XMC_CCU4_SLICE_PRESCALER_64, PWM4_TIMER_PERIOD, DISABLED},     // PWM disabled  33  P3.4
```

**Review by jaenrig-ifx — CHANGES_REQUESTED** [review]
All good beside the 2 suggestions :)

File: variants/XMC4700/config/XMC4700_Relax_Kit/pins_arduino.h (hunk 2 above)
Comment (boramonideep) [reply variants/XMC4700/config/XMC4700_Relax_Kit/pins_arduino.h]: Thanks for point out. Changes have been made.

**Review by jaenrig-ifx — APPROVED** [review]

**Review by djaumann — APPROVED** [review]

**Pull request merged**